lib/biokbase/narrative_method_store/client.py

```diff
@@ -25,8 +25,7 @@
 
 
 def _get_token(user_id, password,
-               auth_svc='https://nexus.api.globusonline.org/goauth/token?' +
-                        'grant_type=client_credentials'):
+               auth_svc="https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login"):
```

Comment:
This is generated code - did you regenerate or edit?

Reply:
Edited. Copied new generated code from NMS